components/EventsList.tsx

```diff
@@ -69,10 +81,10 @@ const styles = StyleSheet.create({
     paddingBottom: 8
   },
   activitiesText: {
-    fontSize: 20,
+    fontSize: 18,
     marginLeft: 16,
     textAlignVertical: "top",
-    fontWeight: "bold"
+    fontFamily: "Open-Sans-Bold"
```

Comment:
@mhayes853  planned to make a separate <Text> component with this styling built-in, sync with him